### ISSUE TITLE
Loading screen seeks through waiting messages insanely fast and never loads the app

### ISSUE THREAD

**Issue by crystall1nedev** [evelyneee/accord#47]
I don't know how else to describe it other than with this:

https://user-images.githubusercontent.com/55281754/195963761-fb4c5084-bf43-4896-a22e-6386bd5e0ca3.mov

Also repeats this message over and over in the terminal (UUID changes, however):

```
[AccordApp:189] Error Domain=NSPOSIXErrorDomain Code=24 "Too many open files" UserInfo={_NSURLErrorFailingURLSessionTaskErrorKey=LocalDataTask <5C3DB051-DA85-4928-B232-FAA2BF4614C2>.<1>, _kCFStreamErrorDomainKey=1, _kCFStreamErrorCodeKey=24, _NSURLErrorRelatedURLSessionTaskErrorKey=(
    "LocalDataTask <5C3DB051-DA85-4928-B232-FAA2BF4614C2>.<1>"
), _NSURLErrorNWPathKey=satisfied (Path is satisfied), interface: en0[802.11], ipv4, ipv6, dns} 
```

And then aborts:
```
Accord(2193,0x700007f70000) malloc: Non-aligned pointer 0x600007929140 being freed (2)
Accord(2193,0x700007f70000) malloc: *** set a breakpoint in malloc_error_break to debug
zsh: abort      /Applications/Accord.app/Contents/MacOS/Accord
```

- Using commit [`e6c5bb5`](https://github.com/evelyneee/accord/commit/e6c5bb5540ca28b826962778b7ee69338014823c), issue not present in [`6d3fc00`](https://github.com/evelyneee/accord/commit/6d3fc0097cc0d0ce107f642d0222880e5151b5fc).
- Be aware that I might just have skill issues

**Comment by evelyneee** [comment]
You're not the only one who had this issue, I'll take a look into it asap

**Comment by RyzenDew** [comment]
also having this issue

**Comment by evelyneee** [comment]
does the latest commit help?

**Comment by RyzenDew** [comment]
> 

yes ci builds are broken and that build works thanks

**Comment by evelyneee** [comment]
This is fixed as of the latest commit a66a336466febc4adeea8c47fbc6b61de84ccef6
